scm/github.go

```diff
@@ -31,32 +31,6 @@ func NewGithubSCMClient(logger *zap.SugaredLogger, token string) *GithubSCM {
 	}
 }
 
-// ListOrganizations implements the SCM interface.
-func (s *GithubSCM) ListOrganizations(ctx context.Context) ([]*pb.Organization, error) {
```

Comment:
Are you sure this is not used by the scm tool? Or by the course creation process. I didn't check.

Reply:
Yes.